test/src/loaders/gtdb/gtdb_taxa_freq_loader_test.py

```diff
@@ -0,0 +1,89 @@
+import inspect
+import os
+import shutil
+import subprocess
+import uuid
+from pathlib import Path
+
+import jsonlines
+import pytest
+
+
+@pytest.fixture
```

Comment:
I think you want to do
```suggestion
@pytest.fixture(scope="module")
```
so this only runs once total, not once per test

Reply:
👍

Reply:
:+1: